src/main/java/org/osiam/resources/provisioning/update/ExtensionUpdater.java

```diff
@@ -113,7 +113,8 @@ private void updateExtensionField(Entry<String, Extension> extensionEntry, UserE
             try {
                 extensionEntitiyField = extensionEntity.getFieldForName(fieldName, true);
             } catch (NoSuchElementException e) {
-                throw new NoSuchElementException("Could not update the extension \"" + urn + "\".", e);
+                LOGGER.info("Update Extension failed: ", e.getMessage());
```

Comment:
shouldn't this be logged on warn level? maybe if this occurred frequently it should be recognized in the logs also when level > info